lib/routes/hypergryph/arknights/arktca.ts

```diff
@@ -0,0 +1,122 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+const rssDescription = `期刊《回归线》 | 泰拉创作者联合会`;
+const Url = 'aneot.arktca.com';
+const Author = `Bendancom`;
```

Comment:
```suggestion
const rssDescription = '期刊《回归线》 | 泰拉创作者联合会';
const Url = 'aneot.arktca.com';
const Author = 'Bendancom';
```
Use template literals only when necessary.